dm/tests/dm_syncer/run.sh

```diff
@@ -71,7 +71,7 @@ function run() {
 	run_dm_syncer $WORK_DIR/syncer2 $WORK_DIR/dm-syncer-2.toml $meta_file --syncer-config-format syncer2
 
 	# wait for dm_syncer to init and start
-	sleep 5
+	sleep 10
```

Comment:
in our doc: `Syncer is no longer maintained. Its features are completely superseded by TiDB Data Migration. It is strongly recommended that you use TiDB Data Migration instead`

just a question do we have plan to delete the code releated to `dm-sycner` from our codebase and `syncer-bin` from `tidb-enterprise-tools`?

also cc @sunzhaoyang